data-esnext.js

```diff
@@ -1407,6 +1407,27 @@ exports.tests = [
     opera10_50: false,
     safari12: false,
   }
+},
+{
+  name: 'Object.hasOwn',
+  category: STAGE3,
+  significance: 'small',
+  spec: 'https://github.com/tc39/proposal-accessible-object-hasownproperty',
+  exec: function () {/*
+    return Object.hasOwn({ x: 2 }, "x") === true;
```

Comment:
This should also test that ToObject is called before ToPropertyKey, since it’s highly likely that impls will just copy over Object.prototype.hasOwnProperty and forget to swap the step ordering.

Reply:
Should I add it as a separate test?

Reply:
Done :+1: